node/pkg/accountant/accountant.go

```diff
@@ -41,26 +41,27 @@ type (
 		BroadcastTxResponseToString(txResp *sdktx.BroadcastTxResponse) string
 	}
 
-	// tokenBridgeKey is the key to the map of token bridges being monitored
-	tokenBridgeKey struct {
+	// emitterKey is the key to a map of emitters to be monitored
+	emitterKey struct {
 		emitterChainId vaa.ChainID
 		emitterAddr    vaa.Address
 	}
 
-	// tokenBridgeEntry is the payload of the map of the token bridges being monitored
-	tokenBridgeEntry struct {
-	}
+	// validEmitters is a set of supported emitter chain / address pairs. The payload is the enforcement flag.
+	validEmitters map[emitterKey]bool
 
 	// pendingEntry is the payload for each pending transfer
 	pendingEntry struct {
-		msg    *common.MessagePublication
-		msgId  string
-		digest string
+		msg         *common.MessagePublication
+		msgId       string
+		digest      string
+		isNTT       bool
```

Comment:
Would it make sense to make this more expandable than just an NTT or token bridge? If this is made more generic then changes like this in the future for new products would probably be easier to expand.

Reply:
Expanding the accountant to more bridges will require extensive changes, so this can be changed if / when that happens.